client/components/tooltip/style.scss

```diff
@@ -73,7 +73,6 @@
 		&__tooltip {
 			// Specific styles for the click tooltip variant.
 			position: relative;
-			max-width: 292px;
```

Comment:
Curious to as to why we moved it from styles to props?

Reply:
This CSS is overridden by `TooltipBase`'s in-line CSS. We might set max-width using CSS class instead of in-line but since it will involve changing `TooltipBase` that is used by other UIs, I created https://github.com/Automattic/woocommerce-payments/issues/8269.